### PR TITLE
PORT-9272 | Change guides to parse files using the file kind

### PR DESCRIPTION
**Pull request by lordsarcastic**
# Description
Changed packages parser to use the GitHub file kind instead of webhooks

## Updated docs pages

- Packages (`docs/build-your-software-catalog/custom-integration/webhook/examples/packages`)

### CODE REVIEW

File: docs/build-your-software-catalog/custom-integration/webhook/examples/packages/trivy.md [hunk 1]
```diff
@@ -3,42 +3,113 @@ sidebar_position: 10
...
+          title: .item.Title
+          blueprint: '"trivyVulnerability"'
+          properties:
+            version: .item.InstalledVersion
+            package_name: .item.PkgName
+            primaryUrl: .item.PrimaryURL
+            description: .item.Description
+            target: .item.Target
+            severity: .item.Severity
+            data_source: .item.DataSource
```
Comment (PeyGis) [inline docs/build-your-software-catalog/custom-integration/webhook/examples/packages/trivy.md:110]: The mapping is not formatted well so it throws error when you copy and paste it in Port. I have modified it so replace it using

```
  - kind: file
    selector:
      query: 'true'
      files:
        - path: '**/results.json'
    port:
      itemsToParse: '[.file.content[] | select(.Vulnerabilities != null) as $input | .Vulnerabilities[] | {VulnerabilityID, PkgName, InstalledVersion, FixedVersion, Title, Description, Severity, References, PrimaryURL, DataSource, Target: $input.Target}]'
      entity:
        mappings:
          identifier: .item.VulnerabilityID
          title: .item.Title
          blueprint: '"trivyVulnerability"'
          properties:
            version: .item.InstalledVersion
            package_name: .item.PkgName
            primaryUrl: .item.PrimaryURL
            description: .item.Description
            target: .item.Target
            severity: .item.Severity
            data_source: .item.DataSource
```

**Review by PeyGis — CHANGES_REQUESTED** [review]
looks good overall. just some few comments

File: docs/build-your-software-catalog/custom-integration/webhook/examples/packages/trivy.md [hunk 2]
```diff
@@ -3,42 +3,113 @@ sidebar_position: 10
...
+</details>
 
-<PythonScript/>
+<details>
+<summary><b>Trivy mapping configuration</b></summary>
+
+```yaml showLineNumbers
+- kind: file
+    selector:
+      query: 'true'
```
Comment (PeyGis) [inline docs/build-your-software-catalog/custom-integration/webhook/examples/packages/trivy.md:93]: this still doesn't work. the kind and selector must be on the same indentation level

**Review by PeyGis — CHANGES_REQUESTED** [review]
check my comment on the indentation for the trivy config mapping

**Review by PeyGis — APPROVED** [review]
LGTM

File: docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/connect-github-codeowners-with-service-team-and-user.md [hunk 3]
```diff
@@ -167,6 +277,15 @@ If you already have the `githubUser`, `githubTeam` and `service` blueprints crea
 
 </details>
 
+<details>
+<summary><b>CODEOWNERS mapping configuration (Click to expand)</b></summary>
+
+```yaml showLineNumbers
+resources:
```
Comment (hadar-co) [inline docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/connect-github-codeowners-with-service-team-and-user.md:284]: this snippet is empty

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**